docs/source/qvm.rst

```diff
@@ -36,9 +36,9 @@ The QVM is available on your local machine. You can initialize a localQVM instan
 
 One executes quantum programs on the QVM using two paradigms: the ``.run(...)`` method, and
 the ``.wavefunction(...)`` method. The former closely mirrors how one will execute programs on a
-real QPU (see `our QPU docs <rigetti.com/qpu>`_, while the latter takes advantage of the virtual machine, and allows
-direct access to the wavefunction. These two methods are described in the following two sections.
-(For information on constructing quantum programs, please refer back to :ref:`basics`.)
+real QPU (check out `our website to see current and legacy QPUs <rigetti.com/qpu>`_, while the latter takes advantage of
+the virtual machine, and allows direct access to the wavefunction. These two methods are described in the following two
```

Comment:
Again, no access to wavefunction, that is now in `WavefunctionSimulator`. "virtual machine" should probably say QVM or virtual quantum computer. I think machine is a little too vague in this context.